docs/config/app-configs.md

```diff
@@ -287,18 +284,32 @@ interface TransformContext {
 }
 ```
 
+### transformHtml
+
+- Type: `(code: string, id: string, ctx: TransformContext) => Awaitable<string | void>`
+
+`transformHtml` is a build hook to transform the content of each page before saving to disk.
+
+::: warning
+Modifying the html content may cause hydration problems in runtime.
```

Comment:
```suggestion
Don't mutate anything inside the `ctx`. Also, modifying the html content may cause hydration problems in runtime.
```

Reply:
I'll check if we can also integrate the head in the client build to solve the hydration missmatch, changes LGTM (I think it can be merged: in the meantime, we can add also the hydration problems to `transformHead`)